<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photo_browse_single</class>
 <widget class="QWidget" name="photo_browse_single">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Label_delete_ps" name="label_delete">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>700</y>
     <width>142</width>
     <height>88</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/tu/Photo/delete.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_return_ps" name="label_return">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>20</y>
     <width>122</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/tu/Photo/return.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_screen" name="label_screen">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>920</width>
     <height>630</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_left_ps" name="label_left">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>298</y>
     <width>54</width>
     <height>205</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/tu/Photo/left.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_right_ps" name="label_right">
   <property name="geometry">
    <rect>
     <x>1176</x>
     <y>298</y>
     <width>54</width>
     <height>205</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/tu/Photo/right.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_full_screen" name="label_full_screen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_full_screen</zorder>
  <zorder>label_b</zorder>
  <zorder>label_delete</zorder>
  <zorder>label_return</zorder>
  <zorder>label_screen</zorder>
  <zorder>label_left</zorder>
  <zorder>label_right</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_screen</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
  <customwidget>
   <class>Label_full_screen</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
  <customwidget>
   <class>Label_return_ps</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
  <customwidget>
   <class>Label_right_ps</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
  <customwidget>
   <class>Label_left_ps</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
  <customwidget>
   <class>Label_delete_ps</class>
   <extends>QLabel</extends>
   <header>photo_browse_single.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
